guides/workspaces.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Workspace organization"
+description: Learn how to organize your Coder workspaces.
+---
+
+This article describes considerations for deciding how to set up your Coder
+workspaces.
+
+In general, the fewer workspaces per developer, the easier it is for the
+individual developer to manage. However, the complexity of the underlying images
+increases as the images need to support multiple projects, each potentially with
+its own language, set of tooling, and dependencies.
+
+## One workspace per developer
+
+With one workspace per developer, you can think of the Coder workspace the way
+you would a laptop: the workspace is where you have all of your languages,
+dependencies, and tooling installed, and it is the one place you'd go to work on
+your projects.
+
+Benefits:
+
+- Fewer workspaces to manage
+- No need to switch between workspaces for different projects
+
+Limitations:
```

Comment:
Actually thinking about this more, maybe instead of Limitations, "Caveats" could be better? These aren't always disadvantages, since they only apply particular cases (hence "can" vs "will")... ideally, users that have very simple workflows will enjoy the benefits and won't be affected by the potential limitations. Maybe "Potential caveats" even?